charts/harvester-cloud-provider/104.0.0+up0.2.3/Chart.yaml

```diff
@@ -0,0 +1,25 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Harvester Cloud Provider
+  catalog.cattle.io/kube-version: '>= 1.23.0-0 < 1.27.0-0'
+  catalog.cattle.io/namespace: kube-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/rancher-version: '>= 2.9.0-0 < 2.10.0-0'
+  catalog.cattle.io/release-name: harvester-cloud-provider
+  catalog.cattle.io/ui-component: harvester-cloud-provider
+  catalog.cattle.io/upstream-version: 0.2.0
+apiVersion: v2
+appVersion: v0.2.0
```

Comment:
This is weird. 
```
appVersion: v0.2.0
version: 104.0.0+up0.2.3
```

Should it not be: `104.0.0+up0.2.0` OR `version: v0.2.3`?

Reply:
It is a mistake and is updated.

Reply:
appVersion is corresponding to the container image version
upstream-version is equal to upstream chart-version

The `harvester-cloud-provider` has different chart-verson and image version.